<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extraStandingOrdersWidget</class>
 <widget class="QGroupBox" name="extraStandingOrdersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>125</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>125</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dauerauftrag Ausführung</string>
  </property>
  <property name="toolTip">
   <string>Ausführungsdaten für den Dauerauftrag</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <property name="title">
   <string>Dauerauftrag Ausführung</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioButton_weekly">
         <property name="text">
          <string>wöchentlich</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup_period</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_monthly">
         <property name="text">
          <string>monatlich</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup_period</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_cycle">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="specialValueText">
        <string>jede</string>
       </property>
       <property name="prefix">
        <string>alle </string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_week_month">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Wochen</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_week_month_2">
       <property name="text">
        <string>am</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_day">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>Donnerstag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-----------</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diese Items</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>werden im</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quelltext</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gelöscht!</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Erstmalig</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Letztmalig</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>nächste Ausführung</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDateEdit" name="dateEdit">
       <property name="displayFormat">
        <string>ddd dd.MM.yyyy</string>
       </property>
       <property name="date">
        <date>
         <year>2011</year>
         <month>8</month>
         <day>23</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="displayFormat">
        <string>ddd dd.MM.yyyy</string>
       </property>
       <property name="date">
        <date>
         <year>2011</year>
         <month>8</month>
         <day>23</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDateEdit" name="dateEdit_3">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>ddd dd.MM.yyyy</string>
       </property>
       <property name="date">
        <date>
         <year>2011</year>
         <month>8</month>
         <day>23</day>
        </date>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_period"/>
 </buttongroups>
</ui>
